.gitignore

```diff
@@ -0,0 +1,4 @@
+bower_components
+node_modules
+npm-debug.log
+bedrock
```

Comment:
Remove `bedrock` from .gitignore. We may choose to ignore it in `bower.json`, but it shouldn't be ignored by git as there are files that should be in the repo in there.